test/paper-menu-button.html

```diff
@@ -55,7 +55,7 @@
 
       setup(function() {
         menuButton = fixture('TrivialMenuButton');
-        trigger = Polymer.dom(menuButton).querySelector('.dropdown-trigger');
+        trigger = Polymer.dom(menuButton).querySelector('[slot="dropdown-trigger"]');
         content = Polymer.dom(menuButton).querySelector('.dropdown-content');
```

Comment:
(`.dropdown-content` -> `[slot="dropdown-content"]`, if you make the change mentioned above.)